src/engine/QueryPlanner.cpp

```diff
@@ -1434,7 +1434,7 @@ std::shared_ptr<ParsedQuery::GraphPattern> QueryPlanner::seedFromPropertyPath(
   AD_THROW(
       ad_semsearch::Exception::NOT_YET_IMPLEMENTED,
       "No implementation for creating a seed from a property path of type " +
-          ((int)path._operation));
+          std::to_string((int)path._operation));
```

Comment:
static_cast<int>() ....  We should have a sanitizer that disallows c-style casts.